mchf-eclipse/hardware/mchf_hw_i2c2.c

```diff
@@ -265,24 +268,29 @@ uint16_t Write_24Cxxseq(uint32_t Addr, uint8_t *buffer, uint16_t length, uint8_t
 	uint16_t retVal = 0xFFFF;
 	count = 0;
 
-	if(Mem_Type == 12)
-		page = 64;
-	if(Mem_Type == 13)
-		page = 32;
-	if(Mem_Type == 14 || Mem_Type == 15)
-		page = 64;
-	if(Mem_Type > 15 && Mem_Type < 19)
-		page = 128;
-	if(Mem_Type == 19)
-		page = 256;
-
```

Comment:
No other Mem_Types like the above are possible so "otherwise" is without any meaning.